onesignal-admin.php

```diff
@@ -490,446 +517,519 @@ function admin_notice_setup_not_complete() {
+                </div>', 86400);
+                    }
+                } else {
+                    if (!empty($response)) {
+                        onesignal_debug('OneSignal API Raw Response:', $response);
+
+                        // API can send a 200 OK even if the notification failed to send
+                        if (isset($response['body'])) {
+                            $response_body = json_decode($response['body'], true);
+
+                            if (isset($response_body['recipients'])) {
+                                $recipient_count = $response_body['recipients'];
+                            } else {
+                                error_log('OneSignal: recipients not set in response body');
+                            }
+
+                            if (isset($response_body['id'])) {
+                                $notification_id = $response_body['id'];
+                            } else {
+                                error_log('OneSignal: notification id not set in response body');
+                            }
+                        } else {
+                            error_log('OneSignal: body not set in HTTP response');
+                        }
+
+                        // updates meta so that recipient count is available for GET request from client
+                        update_post_meta($post->ID, 'recipients', $recipient_count);
+
+                        // updates meta for use in cancelling scheduled notifs
+                        update_post_meta($post->ID, 'notification_id', $notification_id);
+
+                        $sent_or_scheduled = array_key_exists('send_after', $fields) ? 'scheduled' : 'sent';
+                        $config_show_notification_send_status_message = $onesignal_wp_settings['show_notification_send_status_message'] == '1';
+
+                        if ($config_show_notification_send_status_message) {
+                            if ($recipient_count != 0) {
+                                set_transient('onesignal_transient_success', '<div class="components-notice is-success is-dismissible">
+                  <div class="components-notice__content">
+                  <p><strong>OneSignal Push:</strong><em> Successfully '.$sent_or_scheduled.' a notification to '.$recipient_count.' recipients.</em></p>
+                  </div>
+                    </div>', 86400);
+                            } else {
+                                set_transient('onesignal_transient_success', '<div class="updated notice notice-success is-dismissible">
+                        <p><strong>OneSignal Push:</strong><em>There were no recipients. You either 1) have no subscribers yet or 2) you hit the rate-limit. Please try again in three minutes. Learn more: https://bit.ly/2UDplAS </em></p>
```

Comment:
Remove " or 2) you hit the rate-limit. Please try again in three minutes.". There is no longer an issue with 0 receipts on de-deupping now.